packages/dart/pubspec.yaml

```
@@ -1,10 +1,10 @@
 name: parse_server_sdk
-description: The Dart SDK for Parse Platform (https://parseplatform.org)
+description: The Dart SDK to connect to Parse Server. Build your apps faster with Parse Platform, the complete application stack.
 version: 5.1.0
```

I think we need to publish a release for this PR, to get the new score on pub.dev. Alternatively we could not publish a release, merge this as `refactor` and just keep the bad score until the next dart release, but it wouldn't hurt to get a better score right away.

```suggestion
version: 5.2.0
```

---

I don't think we need to give a new version for this PR, only a ```refactor``` will be better

---

Alright

---

Alright